cypress/support/testData.js

```diff
@@ -0,0 +1,9 @@
+export const user = {
+    username: 'AO',
+    password: 'Asdasdasd1!'
+}
+
+export const book = {
+    name: 'Speaking JavaScript',
+    description: 'Like it or not, JavaScript is everywhere these days-from browser to server to mobile-and now you, too, need to learn the language or dive deeper than you have. This concise book guides you into and through JavaScript, written by a veteran programmer who o',
+}
\ No newline at end of file
```

Comment:
Please add a newline at the end of the file for better readability and to prevent potential issues in some tools. 📖